### DISCUSSION TITLE
Problem with parentheses in path

### DISCUSSION THREAD

**Discussion by FedyaSavchuk** [mockoon/mockoon#521] [Help] [answered]
**Case**
In path I use parentheses. This isn't my choice (external api)
Example: localhost:8082/dataSet(someData)

**Problem**
When I send this request in Mockoon I get 404 error, because of Mockoon doesn't support (encode/decode) this symbols

**My bad solution**
Send in Mockoon encoded parentheses, like in this wiki link:
https://en.wikipedia.org/wiki/PC_Tools_%28software%29

But this solution doesn't suit me.
Is there any other way I can solve this problem in Mockoon?

**Comment by 255kb** [comment, marked as answer]
You should be able to solve your problem by escaping the parentheses when declaring the route: 
![image](https://user-images.githubusercontent.com/7489814/126495944-8a7c03fd-b5b8-438f-8cdc-46302b530927.png)

Mockoon is using Express behind the scene, which is considering parentheses as part of regex expressions. 
You can always refer to their [routing documentation](https://expressjs.com/en/guide/routing.html) if needed.

**Reply by FedyaSavchuk** [reply]
Good news! It's working! Thank you :)

**Reply by 255kb** [reply]
Perfect 👍
